app/builtin-pages/views/start.js

```diff
@@ -37,6 +38,8 @@ async function setup () {
     userProfile.title = 'Your profile'
   }
   settings = await beakerBrowser.getSettings()
+  userSetupStatus = await beakerBrowser.getUserSetupStatus()
```

Comment:
Let's not use userSetupStatus for this, because we'll need that later for setting up a profile. Instead, use the technique that `latestVersion` uses in start.js (the siteData DB). Create a new entry, like 'watched-tour'.